src/app/login/view.nim

```diff
@@ -115,6 +115,8 @@ QtObject:
         msg = getCurrentExceptionMsg()
       result = StatusGoError(error: msg).toJson
 
+  proc moveToAppState*(self: LoginView) {.signal.}
+
   proc loginResponseChanged*(self: LoginView, error: string) {.signal.}
```

Comment:
Do we still need this event?

Reply:
I did not check the code (but I should probably)...

Reply:
discussed in Discord.